src/psyclone/tests/gocean1p0_stencil_test.py

```diff
@@ -229,17 +229,16 @@ def test_stencil_case_1():
     '''
```

Comment:
Please could you update the copyright date (and authorship?) of this file.

Reply:
Done.